### DISCUSSION TITLE
Move repository to organizations account

### DISCUSSION THREAD

**Discussion by pedrospeixoto** [pedrospeixoto/MPAS-PXT#12] [Ideas]
Should we move this repository to an organization github account? 
This way, it would not be tied to my personal username, but rather to a general organization name: suggestions? 
- MPAS-Research?
- MONAN-Research?  

Or should we keep it this way until more things are incorporated?

**Comment by guilhermeltm** [comment]
Since this is a project to be developed widely by our research community I think it does make sense to move it to a separate organization account. And seeing MPAS-BR as a project to be eventually integrated into MONAN (although we also use it independently for various research purposes), I like your second suggestion in particular. But isn't there already any such account dedicated to the MONAN project? I found some here (https://github.com/monanadmin and https://github.com/MONAN-MODEL), but I am not sure what their purpose is.

I imagine other people are also working on potential submodels for MONAN, so that at least repositories spread around might exist. Maybe it would be a good idea, if those reps exist, to already centralize them under a unique MONAN account? It would be nice to have a centralized overview of what is being developed within the MONAN context.

**Comment by pedrospeixoto** [comment]
Great points!

MONAN has a repository focused on production code, aiming at operation
needs (it is hosted in the link you mentioned), administrated by INPE.
Development there needs to follow very strict guidelines. On existing
repositories (Both managed by INPE):
https://github.com/monanadmin : hosts preliminary developments and
documentation for future monan codes, like the comparison codes for the
decision for MPAS.
https://github.com/MONAN-MODEL : will host the future monan production codes

The central MONAN account would be administered by INPE, and should have
more strict rules for adding collaborators.

The idea of MPAS-BR is that it is a collaborative research code, with much
more flexibility. Some ideas of the research codes can (and will) be pushed
to MONAN main code, but many things may be solely of research interest and
not operational. For instance, we may have testing options in the codes
(lots of conditionals) that are unwanted in production codes. Our
developments should be open to collaborators, as we can handle better the
risk of someone breaking things. We should also be more "adventurous" and
try odd things out. So I guess we should name it something accordingly. I
like the idea of it been research guided, not production, so
MONAN-Research.

My suggestion would be an "organization" named MONAN-Research, and a
repository named MPAS-BR.

Prof. Dr. Pedro da Silva Peixoto
Associate Professor
Applied Mathematics
University of São Paulo (USP <http://www.usp.br/>)

www.ime.usp.br/~pedrosp


Em sex., 3 de nov. de 2023 às 13:09, guilhermeltm ***@***.***>
escreveu:

**Reply by guilhermeltm** [reply]
OK, so there is no MONAN account for research code. Then I completely agree we should create a new one -- MONAN-Research looks good to me -- and link MPAS-BR to it. 

Maybe it would be nice then to share this idea with other research groups to encourage people to develop their MONAN-related research projects under this same account. My point is only that having the different MONAN research projects centralized under a single account could improve their visibility across groups and help sparking cross-collaborations.

**Reply by favba** [reply]
Can we use the "MONAN" brand without issues? We should probably run by the PI of the project (Is it Saulo Freitas?)

Another option is to follow te MPAS-Dev organization and name ours MPAS-BR-Dev

**Comment by pedrospeixoto** [comment]
I will talk with Saulo Freitas and Pedro Dias. In case there is some
concern on using the MONAN naming or other issues, we can go for
MPAS-BR-Dev, which would be an easy solution. Or we can just call the
organization MPAS-BR and have repositories inside with "dirtier" and
"cleaner" codes.

Prof. Dr. Pedro da Silva Peixoto
Associate Professor
Applied Mathematics
University of São Paulo (USP <http://www.usp.br/>)

www.ime.usp.br/~pedrosp


Em seg., 6 de nov. de 2023 às 14:33, Felipe A. V. de Bragança Alves <
***@***.***> escreveu:
